git-cliff-core/src/commit.rs

```diff
@@ -30,14 +30,15 @@ use serde::{
 	Deserialize,
 	Serialize,
 };
+use serde_json::value::Value;
 
 /// Regular expression for matching SHA1 and a following commit message
 /// separated by a whitespace.
 static SHA1_REGEX: Lazy<Regex> = lazy_regex!(r#"^\b([a-f0-9]{40})\b (.*)$"#);
 
 /// Object representing a link
 #[derive(Debug, Clone, Eq, PartialEq, Deserialize, Serialize)]
-#[serde(rename_all = "camelCase")]
+#[serde(rename_all(serialize = "camelCase"))]
```

Comment:
Why is this required btw?

Reply:
IIRC, it expected camelCase fields in the input JSON before I added this. I'll check if it's really necessary.

Reply:
Yes, looks like it's not necessary for `Link` , but for `Release` and `Commit`, it is.